KeeperSdk/vault/RecordTypeCommands.cs

```diff
@@ -100,6 +101,43 @@ public async Task<string> DeleteRecordTypeAsync(string recordTypeId)
             }
         }
 
+        public async Task<List<string>> LoadRecordTypesAsync(string filePath)
```

Comment:
This method seems to belong to the Commander not SDK.
Is it used inside the SDK?